MotionMark/resources/debug-runner/graph.js

```diff
@@ -334,7 +334,7 @@ Utilities.extendObject(window.benchmarkController, {
                 .domain([0, complexityMax]);
 
 
-        const minFrameRate = this._targetFrameRate / 2;
+        const minFrameRate = this._targetFrameRate / 4;
         const maxFrameRate = this._targetFrameRate * 1.5;
```

Comment:
The same two lines are exist in `benchmarkController.createComplexityGraph()`. Can we these calculations in two separate functions instead:

```
_minFrameRate: function() 
{ 
    return this._targetFrameRate / 4;
}
```